portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,48 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Victor Alvarez - Home</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body>
+    <!-- Navigation bar from W3schools.com -->
+     <ul>
+        <li class="active"><a href="index.html">About Me</a></li>
+        <li><a href="resume.html">Resume</a></li>
+    </ul>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Victor Alvarez's Portfolio</h1>
+
+      <img src="/images/promPhoto.jpeg" alt="image of myself" style="display: block; margin-left: auto; margin-right: auto; height: 50%; width: 50%; border-radius: 50%;"/>
```

Comment:
Commenting on style="display: block; margin-left: auto; margin-right: auto; height: 50%; width: 50%; border-radius: 50%;"

Usually you don't style html tags directly. Instead you create a css file and a css class, and you do a "class=" on the html tag. Looks like you have a css file already made below, so you can add this style there and then reference that new class